package.json

```diff
@@ -66,6 +67,10 @@
     "xtend": "^4.0.0"
   },
   "devDependencies": {
+    "cson": "^3.0.2",
+    "js-yaml": "^3.5.2",
+    "mz": "^2.1.0",
+    "popsicle": "^3.0.1",
     "typings": "^0.2.3"
```

Comment:
This is perfectly fine and thank you :+1: :100: . Just wanted to share that atom does install `devDependencies` on user machines for whatever reason. That's why we had to remove `grunt` / `grunt-ts`. Leaving these in shouldn't be a problem though :rose: Again just sharing past experiences :rose: